test/functional/api/vim_spec.lua

```diff
@@ -3829,5 +3829,11 @@ describe('API', function()
       eq({'aa'}, meths.buf_get_lines(0, 0, 1, false))
       assert_alive()
     end)
+    it("'make' command works when argument count isn't 1 #19696", function()
+      meths.cmd({ cmd = 'make' }, {})
+      assert_alive()
+      meths.cmd({ cmd = 'make', args = { 'foo', 'bar' } }, {})
+      assert_alive()
```

Comment:
This test can potentially be dangerous.

Reply:
Hmm... is there any way to bypass that? I guess we can just remove the test if there's no scope for a regression?

Reply:
needs to set `makeprg` to something innocuous?

Reply:
Oh I guess that's a way, now I feel stupid for not thinking of that

Reply:
Actually there's not really anything I can think of that we can make `makeprg` use honestly... Might just be better to remove the test